CONTRIBUTING.md

```diff
@@ -179,6 +179,7 @@ Our competitive programming lists are not as strict about these exclusions. The
 
 We're not that fussy about book-ness. Here are some attributes that signify that a resource is a book:
 
+
```

Comment:
:warning: As sugested, it will be a great idea to revert whitespaces / blank lines in this file